app/views/users/index.html.erb

```diff
@@ -13,11 +13,9 @@
 
 <% @users.each do |user| %>
   <tr>
-    <td><%= user.name %></td>
+    <td><%= link_to user.name, user_path(user) %></td>
```

Comment:
How about we keep 'Edit' and 'Impersonate', and move the 'Show' link to be on the user's name directly.
"have the Terms#index page just have Edit", do you mean add 'Edit' buttons on Terms#index?